def/de/menu_linux.cson

```diff
@@ -1,133 +1,133 @@
 Menu:
   "&File":
-    value: "File(&F)"
+    value: "Datei"
```

Comment:
Considering [point #3 of CONTRIBUTING.md](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md#translation-guide), we can preserve hotkey hint by appending `(&F)` to enhance UX.

Any other idea is welcome to discuss : )